charts/catalog/values.yaml

```diff
@@ -44,6 +44,9 @@ apiserver:
       globalNamespace: servicecatalog
   # Log level; valid values are in the range 0 - 10
   verbosity: 10
+  # disable authentication so that people don't have to bother setting up certificates
+  # properly just to get the walkthrough working
+  enableAuth: false
```

Comment:
Having written this Helm chart and others... a lesson I have learned is to plan for the future. If you think there will be _more_ authentication-related settings we will want to concern ourselves with in the future (which I think is very likely), we should consider adopting this structure instead:

```
apiserver:
  ...
  auth:
    # Comment...
    enabled: false
```

This allows room for growth without having to make breaking changes or resorting to uglier attributes that would look like `apiserver.authOptionX` and `apiserver.authOptionY`.